core/chains/evm/client/rpc_client.go

```diff
@@ -30,6 +30,47 @@ import (
 	ubig "github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils/big"
 )
 
+//go:generate mockery --quiet --name EvmRpcClient --structname MockEvmRpcClient --filename "mock_evm_rpc_client_test.go" --inpackage --case=underscore
+type EvmRpcClient interface {
```

Comment:
We should try to avoid including parts of package path into type name.
What do you think of renaming this to ChainClientRPC (seems like we only use it there)?
Another option is to rollback change of making RpcClient public and name the interface as RpcClient

Reply:
I think renaming to ChainClientRPC makes sense and is the simplest solution for now.